src/modules/migrations/add.ts

```diff
@@ -59,3 +59,16 @@ export const addMigration = async (params: AddMigrationParams) => {
     `Migration ${migrationName} has been created sucessfully in ${chalk.blue(saveMigrationPath)}`,
   );
 };
+
+const padWithLeadingZeros = (order: number, padWithLeadingZeros: number | undefined) =>
```

Comment:
Nipick: Is the parameter called the same as the function? I am surprised that the compiler does not yell :D (how does it know that you don't want to call that function recursively)... I would rename the parameter